Simplenote/src/main/java/com/automattic/simplenote/utils/AccountVerificationWatcher.java

```diff
@@ -19,6 +16,7 @@
  */
 public class AccountVerificationWatcher implements Bucket.OnNetworkChangeListener<Account> {
     public enum Status {
+        UNKNOWN,
```

Comment:
Nothing that this exposes `UNKNOWN` to the listeners. While it's not a problem to ignore that, how do you feel about introducing `UNKNOWN` and exposing it vs. storing `null` for "unknown" and only exposing the known states?

Reply:
Exahustiveness in pattern matching (if conditions in Java) is important and as you mention, it is not checked in the listener. Let me use `null` which does not affect the rest of the code.

Reply:
Fixed in 5d421ed.